src/draw/sw/helium/lv_blend_helium.h

```diff
@@ -0,0 +1,1347 @@
+/**
+ * @file lv_blend_to_helium.h
```

Comment:
It looks this file should be removed. The actual file used is in draw/sw/blend/helium

Reply:
Solved